internal/entity/marker.go

```diff
@@ -358,7 +358,11 @@ func (m *Marker) SyncSubject(updateRelated bool) (err error) {
 		UpdateColumns(Map{"subj_uid": m.SubjUID, "subj_src": SrcAuto, "marker_review": false}).Error; err != nil {
 		return fmt.Errorf("%s (update related markers)", err)
 	} else if res.RowsAffected > 0 && m.face != nil {
-		log.Debugf("markers: matched %s with %s", subj.SubjName, m.FaceID)
+		subjName := ""
+		if subj != nil {
+			subjName = subj.SubjName
+		}
+		log.Debugf("markers: matched %s with %s", subjName, m.FaceID)
```

Comment:
Based on the code above, subj should never be nil here. However, I've added a Subject.String() function which can handle nil values and also makes the code easier to read, see https://github.com/photoprism/photoprism/commit/fc5d8cb9a9c5e4254ddc08bbcf3dee426505d0c7.